lib/spack/spack/cmd/install.py

```diff
@@ -72,6 +72,9 @@ def setup_parser(subparser):
     subparser.add_argument(
         '--fake', action='store_true', dest='fake',
         help="fake install. just remove prefix and create a fake file")
+    subparser.add_argument(
+        '--force', '-f', action='store_true', dest='force',
```

Comment:
If you're adding a new flag, can you update the tab-completion for `spack install`? See:
https://github.com/LLNL/spack/blob/develop/share/spack/spack-completion.bash#L408